<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="vehicle_model" description="vehicle model name"/>
  <arg name="use_sim_time" default="false" description="use_sim_time"/>

  <!-- Global parameters -->
  <group scoped="false">
    <include file="$(find-pkg-share global_parameter_loader)/launch/global_params.launch.py">
      <arg name="use_sim_time" value="$(var use_sim_time)"/>
      <arg name="vehicle_model" value="$(var vehicle_model)"/>
    </include>
  </group>

  <!-- Rviz -->
  <group>
    <node pkg="rviz2" exec="rviz2" name="rviz2" output="screen" args="-d $(find-pkg-share obstacle_avoidance_planner)/rviz/obstacle_avoidance_planner.rviz"/>
  </group>
</launch>
